<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>aboutDialog</class>
  <widget class="QDialog" name="aboutDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>500</width>
        <height>760</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>About lignumCAD</string>
    </property>
    <property name="windowIcon">
      <pixmap>lignumCAD.png</pixmap>
    </property>
    <property name="sizeGripEnabled">
      <bool>false</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>6</number>
      </property>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <spacer name="Spacer7">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QLabel" name="PixmapLabel1">
              <property name="pixmap">
                <pixmap>splash.png</pixmap>
              </property>
              <property name="scaledContents">
                <bool>true</bool>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer8">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QLabel" name="programVersionLabel">
          <property name="font">
            <font>
              <family>Arial [Xft]</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
            </font>
          </property>
          <property name="text">
            <string>lignumCAD Version x.y</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="copyrightLabel">
          <property name="font">
            <font>
              <family>Arial [Xft]</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
            </font>
          </property>
          <property name="text">
            <string>Copyright Â© 2002 lignum Computing, Inc.</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="contactLabel">
          <property name="font">
            <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
            </font>
          </property>
          <property name="text">
            <string>www.lignumcomputing.com
e-mail: lignumcad@lignumcomputing.com</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="GPLLabel">
          <property name="text">
            <string>This program is licensed to you under the terms of the GNU General Public License Version 2 as published by the Free Software Foundation. This gives you legal permission to copy, distribute and/or modify this software under certain conditions. For details, see the file 'COPYING' that came with this software distribution. If you did not get the file, send email to lignumcad@lignumcomputing.com.

The program is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="Line" name="Line1">
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QGridLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item row="2" column="1">
            <widget class="QLabel" name="freeTypeVersionLabel">
              <property name="text">
                <string>Version x.y.z</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="openCASCADELabel">
              <property name="text">
                <string>OpenCASCADE &lt;www.opencascde.org&gt;:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="0" rowspan="1" colspan="2">
            <widget class="QLabel" name="acknowledgementsLabel">
              <property name="text">
                <string>lignum Computing, Inc. is pleased to acknowledge the following Open Source components of lignumCAD:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item row="4" column="1">
            <widget class="QLabel" name="openGLGLUVersionLabel">
              <property name="text">
                <string>Version x.y.z</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="3" column="0">
            <widget class="QLabel" name="qtLabel">
              <property name="text">
                <string>Qt &lt;www.trolltech.com&gt;:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QLabel" name="openCASCADEVersionLabel">
              <property name="text">
                <string>Version x.y.z</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QLabel" name="freeTypeLabel">
              <property name="text">
                <string>FreeType &lt;www.freetype.org&gt;:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="4" column="0">
            <widget class="QLabel" name="openGLGLULabel">
              <property name="text">
                <string>OpenGL GLU &lt;www.sgi.com&gt;:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="3" column="1">
            <widget class="QLabel" name="qtVersionLabel">
              <property name="text">
                <string>Version x.y.z</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Line" name="Line1_2">
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="openGLLabel">
              <property name="text">
                <string>OpenGL Driver</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLabel" name="openGLVersionLabel">
              <property name="text">
                <string>Version x.y.z</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <spacer name="Horizontal Spacing2">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonOk">
              <property name="text">
                <string>OK</string>
              </property>
              <property name="shortcut">
                <number>0</number>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer6">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>aboutDialog</receiver>
      <slot>accept()</slot>
    </connection>
  </connections>
</ui>
